src/configManager/helpers/validators.js

```diff
@@ -0,0 +1,16 @@
+module.exports.validateBenchmarkWeights = (config) => {
```

Comment:
better to be middleware
you can see that in many routes we do middleware which do validations before entering the controller/model